lib/utils/dynamic-require.js

```diff
@@ -33,3 +35,14 @@ function dynamicRequireSync({ path, backupPath, deleteCache = false }) {
 
   return require(path);
```

Comment:
I think we could avoid using `fs.existsSync`, and possibly removing the two async/sync split in this file, if we use `require` to try to load the file and then respond if that effort fails. If `error.code === 'MODULE_NOT_FOUND'`, then we know the same thing that we previously used `fs.existsSync` to discover; but if there is no error, we've already loaded the module and are ready to go without another fs or require call.

Reply:
This makes sense, thanks for the idea.